package.json

```diff
@@ -13,15 +13,15 @@
     "prepare": "npm run compile"
   },
   "author": "MultiversX",
-  "license": "GPL-3.0-or-later",
+  "license": "MIT",
   "devDependencies": {
-    "@multiversx/sdk-core": ">= 13.17.1",
-    "@multiversx/sdk-web-wallet-cross-window-provider": ">= 2.0.2",
+    "@multiversx/sdk-core": "^14.x",
```

Comment:
Usually, when a new major release is used like in this case, we have to increase also the consumer package version, then would be necessary to increase the webview-provider package major version.

Reply:
Also, it should be used fixed version for all packages